### ISSUE TITLE
[docs]: Performance

### ISSUE THREAD

**Issue by evilebottnawi** [rehypejs/rehype#40]
### Subject of the feature

Will be great to look at `Performance` compassion with other HTML parsers like here https://github.com/fb55/htmlparser2#performance

### Expected behavior

There are not problems. I think a table like this would help increase popularity. And let the developers get some metrics.

### Alternatives

I think we can use https://github.com/AndreasMadsen/htmlparser-benchmark to get results.

### Why?

We want to integrate `rehype` in webpack and webpack ecosystem to handle HTML/HTML entrypoints. We evaluate existing solutions and their convenience. The project looks very good and has everything we need (API).

**Comment by wooorm** [comment]
Thanks for the compliments!

I used to put performance on a couple of readmes before, but as *correct*ness is the most important selling point, found them not very valuable. E.g., for webpack you’re also looking at API, correctness, and ASTs probably, right?

I’m not opposed to this though!

**Comment by evilebottnawi** [comment]
> I used to put performance on a couple of readmes before, but as correctness is the most important selling point, found them not very valuable. E.g., for webpack you’re also looking at API, correctness, and ASTs probably, right?

Yes, and performance too :smile: Anyway API/AST and correctness is preferable. Performance can be improved in any time
